protocol/createacls/createacls.go

```diff
@@ -21,29 +21,37 @@ func (r *Request) Broker(cluster protocol.Cluster) (protocol.Broker, error) {
 }
 
 type RequestACLs struct {
-	ResourceType        int8   `kafka:"min=v0,max=v2"`
-	ResourceName        string `kafka:"min=v0,max=v2"`
-	ResourcePatternType int8   `kafka:"min=v0,max=v2"`
-	Principal           string `kafka:"min=v0,max=v2"`
-	Host                string `kafka:"min=v0,max=v2"`
-	Operation           int8   `kafka:"min=v0,max=v2"`
-	PermissionType      int8   `kafka:"min=v0,max=v2"`
+	// We need at least one tagged field to indicate that v2+ uses "flexible"
```

Comment:
Could you add a test for these using prototest?

Reply:
Good call, I just added these